AutonomousDemo/DeterminePath.cs

```diff
@@ -8,20 +8,33 @@ namespace AutonomousDemo
 {
     public class DeterminePath
     {
-        public void CalculateTravelTime(double meters, Road road, VehicleMotion vehicleMotion)
-        {
+        //these don't belong here
+        VehicleMotion vehicleMotion = new VehicleMotion(new Velocity(0));
+        DeterminePath determinePath = new DeterminePath();
+        Road road = new Road(27);
 
-            if (vehicleMotion.CurrentVelocity.Speed == 0)
+        Velocity displacement = new Velocity(3.4);
+        
+        public Velocity CalculateDriveVelocity()
+        {
+            if (vehicleMotion.CurrentVelocity.Speed < road.SpeedLimit)
             {
-                var accelerationTime = 4.0;
-                var constantAccelerationDistancePerSecond = road.SpeedLimit/accelerationTime; //travel this many meters per second
-
-                var nonAcceleratingdistance = meters - road.SpeedLimit;
-                var nonAcceleratingTime = nonAcceleratingdistance / road.SpeedLimit;
-                var totalTime = accelerationTime + nonAcceleratingTime;
-
-                //    vehicleMotion.ConstantAcceleration(travelTime, finalVelocity, vehicleMotion.CurrentVelocity);
+                vehicleMotion.DistanceTraveled += displacement.Speed;
+                return vehicleMotion.CurrentVelocity + displacement;
```

Comment:
On second thought (as I look over the whole file), I was using the displacement velocity to adjust current velocity, speeding up and slowing down by 3.4 m/s.  What would be more ideal?